docs/quickstart.asciidoc

```diff
@@ -29,18 +29,17 @@ include::supported-versions.asciidoc[]
 [id="{p}-deploy-eck"]
 == Deploy ECK in your Kubernetes cluster
 
-IMPORTANT: Check the <<{p}-upgrading-eck,upgrade notes>> first if you are attempting to upgrade an existing ECK deployment.
+Things to consider before you start:
 
-[NOTE]
-====
+* For this quickstart guide, your Kubernetes cluster is assumed to be already up and running, and conform to one of the link:k8s-quickstart.html[supported versions].
```

Comment:
I think a cluster cannot conform to a version. It is running a certain version maybe?

Reply:
Something like that:

> For this quickstart guide, your Kubernetes cluster is assumed to be already up and running. Before you proceed with the ECK installation, make sure you check the link:k8s-quickstart.html[supported versions].